contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+
+Thresholds
+----------
+As described in the [`Incentives system for Swarm <https://swarm-gateways.net/bzz:/swarm.eth/ethersphere/orange-papers/1/sw%5E3.pdf>`_], this accounting system works by allowing the mutual accounting to oscillate in a defined range. This range is delimited by two thresholds:
+* The payment threshold is a number at which if the node goes below that relative to a peer, it should trigger a payment to that peer (a cheque, see below). This is initiated by the debitor node.
+* The disconnect threshold is a number at which if the node goes above that number relative to a peer, it can (it's up to the peer but this is the expected behavior) disconnect the peer. This is monitored by the creditor.
+
+The disconnect threshold amount should be bigger than the payment threshold in absolute terms.
+
+Fraud risks
+-----------
+Of course, the design with individual peer databases means that nodes can alter their database and pretend to have different balances to other nodes. The simplicity of this mutual accounting though effectively significantly limits fraud, as if node A modifies its entry with B, at settlement time, B will verify in its local database that the claim is not matched with its records and simply ignores fraudulent claims. Normal behavior is to disconnect a node in this case. However, we want to make it clear that a node can lose funds up to the disconnect threshold amount due to freeriders.
+
+.. note::
+
+   A peer can be freeriding by consuming resources up to the disconnect threshold. At this point, if there is no settlement, the peer simply gets disconnected.
+
+
+Imbalances
+----------
+Imbalances between nodes more generally leads to disconnects from peers. The downside of this is that if node A was able to send an accounted message to B, which successfully left A, but for some reason never arrived at B, then this would lead to imbalances as well. Swarm currently treats this case as an edge case and does not implement any balance synchronization nor clearance protocol to address such cases. It may though be considered for the future.
+
+Settlement with cheques
+------------------------
+The balance entries for each node in local databases represent just accounting entries in Swarm's internal accounting unit, but are just numbers. The Swarm papers document the notion of a threshold at which finally a financial settlement protocol is initiated. 
+
+If a node's balance with a peer crosses the payment threshold, which is a number every node can set individually (see `swap-payment-threshold` flag), but has a reasonable default defined in the code, then the node kicks off the settlement process. This process involves a series of security and sanity checks, culminating in sending a **signed cheque** to its peer. This signed cheque is a piece of data containing the amount, the source chequebook address and the beneficiary chequebook address, as well as the signature of the cheque issuer. 
```

Comment:
The cheque contains the beneficiary address, not the beneficiary chequebook address.